challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
 <!-- Answer Starts Here -->
-
+Absolute path -- starts at "Users" so you know the computer will recognize it wherever you are.
 <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
 <!-- Answer Starts Here -->
-
+Absolute, so others can access it wherever they are, whatever machine they're on.
 <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
 <!-- Answer Starts Here -->
-
+Relative, if it's internal to your project than you can access with a relative path.
```

Comment:
You are correct about using a relative path. However, we can always refer to anything with a relative path; the reason why we want to use a relative path is that the image, being part of the project, will move when the project moves, so the absolute paths for the image and for the project itself will change together but keep the same relationship.